Formula/w/weaviate.rb

```diff
@@ -0,0 +1,18 @@
+class Weaviate < Formula
+  desc "Open-source vector database that stores both objects and vectors"
+  homepage "https://weaviate.io/developers/weaviate/"
+  url "https://github.com/weaviate/weaviate/archive/refs/tags/v1.27.6.tar.gz"
+  sha256 "d8cad0339ccfd081be6afe1afd879f64bba224ed7ecbdb8b974853b536584330"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "./cmd/weaviate-server"
+    bin.install "weaviate-server" => "weaviate"
+  end
+
+  test do
+    assert_match '"version":"1.27.6"', shell_output("#{HOMEBREW_PREFIX}/bin/weaviate 2>&1", 1)
```

Comment:
can you do a server start test? there are examples with other formulae.

Reply:
Sure, I changed the test to do a server test and the version is now checked with the REST API (one stone two birds)